src/utils/js_utils.ml

```diff
@@ -1201,7 +1201,11 @@ let parse_fragment () =
 let set_fragment args =
   let pairs = List.map (fun (n, v) -> n ^ "=" ^ v) args in
   let fragment = String.concat "&" pairs in
-  Url.Current.set_fragment fragment
+  (* TODO: Replace with
+  {| Url.Current.set_fragment fragment |}
+  when https://github.com/ocsigen/js_of_ocaml/pull/1497 is shipped
```

Comment:
Did you create an issue to track this down?

Reply:
There is already https://github.com/ocaml-sf/learn-ocaml/issues/539

→ do you suggest that we create yet another issue, more to the point?
→ otherwise, we can just keep #539 and not close it

Reply:
@yurug Anyway, I'm afraid we will wait for a (very) long time before we can do this very small replacement,
because we should wait that js_of_ocaml ships this fix, and that the newest js_of_ocaml release becomes one day the minimal required version in learn-ocaml.

so, not sure it is worth an issue.

WDYT?